lib/flipper/adapters/http.rb

```diff
@@ -115,8 +112,20 @@ def add(feature)
       end
 
       def get_multi(features)
```

Comment:
This should Just Work™ when https://github.com/jnunemaker/flipper/pull/224 is merged.